### ISSUE TITLE
Regular Expressions: Positive and Negative Lookahead

### ISSUE THREAD

**Issue by Setarii** [freeCodeCamp/freeCodeCamp#39365]
**Describe your problem and how to reproduce it:**
This solution requires GROUPING which we learn AFTER this lesson. This needs to be adjusted to require an answer that does NOT REQUIRE GROUPING.

**Add a Link to the page with the problem:**
https://www.freecodecamp.org/learn/javascript-algorithms-and-data-structures/regular-expressions/positive-and-negative-lookahead

**Tell us about your browser and operating system:**
* Browser Name: Doesn't matter
* Browser Version: Doesn't matter
* Operating System: Doesn't matter
![fcc require grouping](https://user-images.githubusercontent.com/13278272/89255135-93f93a80-d5e6-11ea-8a8c-28eccc0d52df.png)

**If possible, add a screenshot here (you can drag and drop, png, jpg, gif, etc. in this box):**

**Comment by nhcarrigan** [comment]
Hello~!

My apologies, but I do not see grouping in the solution you have shared. As stated in the instructions, a positive lookahead uses the syntax `(?=...)`, where the `...` is replaced with the characters you want to match. In the following lesson, grouping is taught - grouping takes the syntax `(set1|set2)`, where you replace `set1` and `set2` with the two alternative character sets you want to match. 

The solution uses lookaheads as explained in this lesson, but does not appear to use grouping. 🙂

**Comment by Setarii** [comment]
Is `()` not a capture group? I may be missing something here. There was another issue with a much earlier lesson that apparently required parenthesis to group commands together that I also did not understand the solution of, because I did not know what the parenthesis even did, or that I was supposed to use them.

**Comment by nhcarrigan** [comment]
Going to copy the relevant instruction from this challenge so I can reference it more quickly:
> A positive lookahead will look to make sure the element in the search pattern is there, but won't actually match it. A positive lookahead is used as `(?=...)` where the `...` is the required part that is not matched.

There are a few different use-cases for parentheses in a regex, and lookaheads are one of them. The text I copied from the lesson instructions reads clearly to me - but my perspective is certainly not the only one. Is there a way you think these instructions could be worded more clearly? 🙂

**Comment by Setarii** [comment]
Hmmm, no I suppose not in this specific case. But I was only confused because of the solution to this problem: https://forum.freecodecamp.org/t/freecodecamp-challenge-guide-restrict-possible-usernames/301363 and this forum thread: https://forum.freecodecamp.org/t/restrict-possible-usernames-solution-requires-knowledge-not-introduced/395660, which, now that I've gotten a bit farther in the lessons, quantity specifiers are not parenthesis, but that was the first solution I had seen with them, so I associated it to "grouping". Either way, I suppose the issue I linked here is solvable without quantity specifiers judging by Solution 1.

Anyway, now I know parenthesis can be actual syntax of regex and not just grouping regex together for various purposes, apologies!
